test/e2e/dse_test.go

```diff
@@ -20,4 +21,9 @@ func createSingleDseDatacenterCluster(t *testing.T, ctx context.Context, namespa
 	dcKey := framework.ClusterKey{K8sContext: f.DataPlaneContexts[0], NamespacedName: types.NamespacedName{Namespace: namespace, Name: "dc1"}}
 	checkDatacenterReady(t, ctx, dcKey, f)
 	assertCassandraDatacenterK8cStatusReady(ctx, t, f, kcKey, dcKey.Name)
+
+	t.Log("Check that we can communicate through CQL with DSE")
+	_, err = f.ExecuteCql(ctx, f.DataPlaneContexts[0], namespace, k8ssandra.SanitizedName(), DcPrefix(t, f, dcKey)+"-default-sts-0",
```

Comment:
This is an expensive CQL command, couldn't we just do a simple `SELECT * FROM system.peers`?

Reply:
Done